series/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from series import views
+
+urlpatterns = [
+    path('', views.SearchSeriesView.as_view(), name='search-series'),
```

Comment:
Use "search_series" because "_" is already used in the file names.

Reply:
Fine, I'll change that. :)

Reply:
@BenoitECP Updated in new version. I also added a "conventions" section in the README to document conventions like this one.